sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -823,33 +766,6 @@ public void sensor_for_sonar_lint_doesnt_provide_metrics() {
     assertNoMeasure(context, componentKey, CoreMetrics.FUNCTIONS);
   }
 
-  @After
-  public void tearDown() {
-    tempReportFiles.forEach(File::delete);
-  }
```

Comment:
I see you didn't put back this file cleaning after, but I don't get why.
Did anything change? Or maybe it didn't make much sense before?